src/popover.ts

```diff
@@ -284,22 +300,19 @@ export function apply() {
     }
     const invoker = target.closest(popoverInvokerSelector);
     const popoverTargetElement = handlePopoverTargetElementInvocation(invoker);
-    for (const popover of [...popovers]) {
-      if (
-        popover.matches('[popover="" i].\\:open, [popover=auto i].\\:open') &&
-        popover !== popoverTargetElement &&
-        !queryAncestorAll(target, '[popover]').includes(popover)
-      ) {
-        popover.hidePopover();
-      }
-    }
+    dismissAutoPopovers(event, popoverTargetElement);
   };
 
-  const addOnClickEventListener = (
-    (callback: (event: Event) => void) => (root: Document | ShadowRoot) => {
-      root.addEventListener('click', callback);
+  const onKeydown = (event: Event) => {
+    if ((event as KeyboardEvent).key === 'Escape') {
```

Comment:
I'm in the habit of also checking for `'Esc'` in cases like this, but that's probably outdated enough by now that we don't need to worry about it?

Reply:
I think `'Esc'` will only be output in IE/EdgeHTML browsers, which I don't think will run this code for other reasons. Having said that I don't mind adding it in with a view to later support those browsers if that's a desire.